packages/smarthr-ui/src/components/Dropdown/DropdownTrigger.tsx

```diff
@@ -13,16 +15,34 @@ import { tv } from 'tailwind-variants'
 
 import { tabbable } from '../../libs/tabbable'
 import { includeDisabledTrigger } from '../../libs/util'
+import { Tooltip } from '../Tooltip'
 
 import { DropdownContext } from './Dropdown'
 
-type Props = PropsWithChildren<ComponentProps<'div'>>
+type ConditionalWrapperProps = {
+  shouldWrapContent: boolean
+  wrapper: (children: ReactNode) => JSX.Element
```

Comment:
```suggestion
  wrapper: FC<PropsWithChildren>
```
とかだとだめなんでしたっけ (JSX.Elementを返すようにしているとTypeScriptのv4.4以前でエラーになる傾向があるのでなるべく壊れにくいコードにする提案のつもりで書いています！

Reply:
@Qs-F 
型修正してみました
https://github.com/kufu/smarthr-ui/pull/5291/commits/41489c6fc38cb1de5cf0a23d2cdd965e87d39307

Reply:
よさそう！